### ISSUE TITLE
1.19 Minecraft clients cannot join the proxy

### ISSUE THREAD

**Issue by KaiKikuchi** [WaterfallMC/Waterfall#744]
Commit https://github.com/PaperMC/Waterfall/commit/9b0080a3ced33d7328837f6f564483ce8208d8e7

This is what shows up on the proxy's log when a 1.19 client tries to join:
```
[19:45:46 INFO]: [/127.0.0.1:63051] <-> InitialHandler has connected
[19:45:46 WARN]: [/127.0.0.1:63051] <-> InitialHandler - corrupted frame: A packet could not be decoded because it was too large. For more information, launch Waterfall with -Dwaterfall.packet-decode-logging=true
>
```

This is the error after enabling the packet decode logging:
```
[19:49:00 INFO]: [/127.0.0.1:63241] <-> InitialHandler has connected
[19:49:00 WARN]: [/127.0.0.1:63241] <-> InitialHandler - corrupted frame: Packet class net.md_5.bungee.protocol.packet.LoginRequest 0 Protocol 759 was too big (expected 65 bytes, got 829 bytes)
>
```

Nothing shows up on the Minecraft server log.

The Minecraft server can be joined directly just fine, and it can be joined with a 1.18 client as well.

EDIT: forgot to mention that the proxy's "online_mode" is set to true.
EDIT2: setting "online_mode" to false shows the same error.

**Comment by KaiKikuchi** [comment]
Sorry I actually tested the same setup with BungeeCord and I get a similar error: 
```
[KaiNoMood] <-> ServerConnector [lobby] - encountered exception: net.md_5.bungee.util.QuietException: Unexpected packet received during server login process!
4b00
```

So I guess this error is not about Waterfall

**Comment by electronicboy** [comment]
Latest builds should be working
<img width="1118" alt="image" src="https://user-images.githubusercontent.com/1228900/172454338-0c3aaf9e-94f0-494f-b983-8ad88de3b05f.png">

**Comment by KaiKikuchi** [comment]
Yes actually I was testing using a Minecraft 1.18.2 server with ViaVersion (I should have mentioned this, my bad!). While the server can be joined directly with that just fine, it cannot be joined through the proxy. So I guess I will have to redirect this issue to the ViaVersion team!

Thanks for the help!

**Comment by NoahvdAa** [comment]
This was an issue with Waterfall and is fixed in the latest build, it's not a viaversion issue.

**Comment by KaiKikuchi** [comment]
Okay thanks for clarifying it! I will test the latest commit again!

**Comment by KaiKikuchi** [comment]
While the issue in my original post was fixed, now I get the same error I get on BungeeCord while using ViaVersion!

I can join a spigot 1.19 server using the proxy with a 1.19 client, but I cannot join a 1.8.8 spigot server with ViaVersion 4.3.0 using the proxy. I can join the same 1.8.8 spigot server with ViaVersion 4.3.0 directly, without the proxy. I can also join the same 1.8.8 spigot server with any client between 1.8.9 and 1.18.2 with or without the proxy.

Is this a proxy issue, or should I report it to the ViaVersion team?

Thanks a lot!

**Comment by electronicboy** [comment]
if the misnomer is between a real 1.19 server and via, that's probably it; All we can tell you is that you got an unexpected packet received by the proxy from the server

**Comment by JCAguilera** [comment]
I'm getting the same error. Using ViaVersion 4.3.0 with a 1.17.1 PaperMC server and latest Waterfall build available in the website (#488). @KaiKikuchi have you found the solution?

**Comment by Packsolite** [comment]
Can confirm, not fixed in latest build with viaversion 4.3.0.

**Comment by KaiKikuchi** [comment]
https://github.com/ViaVersion/ViaVersion/issues/2918
